### PR TITLE
fix(ToggleControl): shrink height, draw on top.

### PR DESCRIPTION
**Pull request by aronhelser**
Bump z-index so it draws on top of parallel coords.
Remove full-height calc, and add a min-height style, so
content resizes naturally.

@jourdain is this what you were thinking? Works for the examples and for divvy.

### CODE REVIEW

File: src/Component/Native/ToggleControl/index.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default class CompositeControlContainer {
       const controlWidth = width < (this.targetWidth + 20) ? (width - 20) : this.targetWidth;
 
       controlDiv.style.width = `${controlWidth}px`;
-      controlDiv.style.height = `${height - 45}px`;
+      // controlDiv.style.height = `${height - 45}px`;
```
Comment (jourdain) [inline src/Component/Native/ToggleControl/index.js]:
```
controlDiv.style.maxHeight = `${height - 45}px`
controlDiv.style.overflow = 'auto'
```

File: style/ComponentNative/ToggleControl.mcss [hunk 2]
```diff
@@ -21,7 +21,7 @@
 }
 
 .control {
-  z-index: 1;
+  z-index: 10;
```
Comment (jourdain) [inline style/ComponentNative/ToggleControl.mcss]: yes

File: style/ComponentNative/ToggleControl.mcss [hunk 3]
```diff
@@ -46,4 +46,5 @@
   display: none;
   position: relative;
   border-top: 1px solid black;
+  min-height: 100px;
```
Comment (jourdain) [inline style/ComponentNative/ToggleControl.mcss]: Do you get the panel to grow based on its content?
I would even go lower like 1rem

**Comment by aronhelser** [comment]
K, all fixed up. I couldn't figure out how to use both the spacer and the background color components in the example.

**Pull request merged**